src/wg/color/package.d

```diff
@@ -1,3 +1,167 @@
 module wg.color.color;
 
-// basic RGB colours...
+import wg.color.rgb;
+
+
+/** 24 bit RGB color type with 8 bits per channel. */
+alias RGB8 =    RGB!("rgb");
+/** 32 bit RGB color type with 8 bits per channel. */
+alias RGBX8 =   RGB!("rgbx");
+/** 32 bit RGB + alpha color type with 8 bits per channel. */
+alias RGBA8 =   RGB!("rgba");
+
+/** Floating point RGB + alpha color type. */
+alias RGBAf32 = RGB!("rgba_f32_f32_f32_f32");
+
+
+/** Set of colors defined by X11, adopted by the W3C, SVG, and other popular libraries. */
+enum Colors
+{
+    aliceBlue            = RGB8(240,248,255), /// <font color=aliceBlue>&#x25FC;</font>
```

Comment:
Should we have a template that would allow us compile time color string parsing so we can define constants like this as well: `RGB8!("F0F8FF")`?

Reply:
@igor84 Yes, I have the function you seek on my other color library... I'll pull it over at some point.
Gotta be able to give hex strings and web colors.